packages/styled-components/package.json

```diff
@@ -17,7 +17,7 @@
     "url": "https://github.com/xing/hops.git"
   },
   "dependencies": {
-    "babel-plugin-styled-components": "^1.5.1",
+    "babel-plugin-styled-components": "1.5.1",
```

Comment:
Can we use `1.6.1` now? Or should we wait for your PR to get merged and then go to the next version?

Reply:
I'd wait as 1.6.1 is still rather slow